src/Tempest/Database/src/Id.php

```diff
@@ -15,11 +15,18 @@
 
     public function __construct(string|int|self $id)
```

Comment:
Is there a reason we don't separate these out?

Reply:
The idea is that all IDs are represented by a single value object, and that people don't have to worry about using UUIDs or normal ids. BUT that's in theory, I haven't really touched this code in ages. But outside the scope of this PR